recipes/cctz/all/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+if (WIN32 AND BUILD_SHARED_LIBS)
+    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

Comment:
this requires at least cmake 3.4. What is the minimum cmake version of this library?

Reply:
CMake 2.8.12

Not sure what to do about this one. Do we need this module definition file? https://cmake.org/cmake/help/v3.4/prop_tgt/WINDOWS_EXPORT_ALL_SYMBOLS.html#prop_tgt:WINDOWS_EXPORT_ALL_SYMBOLS

Reply:
That module definition file is the one storing the symbols for windows libs. CMake can generate it with that flag, but it is only supported since CMake 3.4. I guess the library does not provide any other way to build it as shared in windows, right?

Reply:
I'm not sure the readme doesn't contain much information about this, if the readme is complete this might not even be necessary? :thinking: 

https://github.com/google/cctz#getting-started

Reply:
Seems it has native CMake support for build type, so it could work without this. Give it a shot!

Reply:
It didn't work without for me